### PR TITLE
GG-21551 Fixed method names for RollingUpgradeStatus and RollingUpgra…

### PR DESCRIPTION
**Pull request by sk0x50**
…deModeChangeResult

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    private static final long serialVersionUID = 0L;
+
+    /** Rolling upgrade status. */
+    private VisorRollingUpgradeStatus status;
+
+    /** Overall status. */
+    private RollingUpgradeModeChangeResult.Result res;
+
+    /** The reason why the operation was failed. */
+    private Exception cause;
```
Comment (vsisko) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java]: Use VisorExceptionWrapper for wrapping of exceptions.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        return supportedFeatures;
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void writeExternalData(ObjectOutput out) throws IOException {
+        out.writeBoolean(enabled);
+        out.writeBoolean(forcedModeEnabled);
+        U.writeString(out, initVer);
+        U.writeString(out, targetVer);
+        out.writeObject(supportedFeatures);
```
Comment (vsisko) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java]: U.writeCollection can be used for collection object.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void readExternalData(byte protoVer, ObjectInput in)
+        throws IOException, ClassNotFoundException {
+        enabled = in.readBoolean();
+        forcedModeEnabled = in.readBoolean();
+        initVer = U.readString(in);
+        targetVer = U.readString(in);
+        supportedFeatures = (Set)in.readObject();
```
Comment (vsisko) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java]: U.readSet can be used for Set object.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatusResult.java [hunk 4]
```diff
@@ -88,21 +89,26 @@ public RollingUpgradeStatus status() {
...
+        return updatedNodes;
     }
 
     /** {@inheritDoc} */
     @Override protected void writeExternalData(ObjectOutput out) throws IOException {
         out.writeObject(status);
-        out.writeObject(initVerNodes);
-        out.writeObject(updateVerNodes);
+        out.writeObject(initNodes);
+        out.writeObject(updatedNodes);
```
Comment (vsisko) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatusResult.java]: U.writeCollection can be used for collection object.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatusResult.java [hunk 5]
```diff
@@ -88,21 +89,26 @@ public RollingUpgradeStatus status() {
...
 
     /** {@inheritDoc} */
     @Override protected void readExternalData(byte protoVer, ObjectInput in) throws IOException, ClassNotFoundException {
-        status = (RollingUpgradeStatus)in.readObject();
-        initVerNodes = (List<String>)in.readObject();
-        updateVerNodes = (List<String>)in.readObject();
+        status = (VisorRollingUpgradeStatus)in.readObject();
+        initNodes = (List<String>)in.readObject();
+        updatedNodes = (List<String>)in.readObject();
+    }
```
Comment (vsisko) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatusResult.java]: U.readList can be used for List object.

**Review by vsisko — CHANGES_REQUESTED** [review]

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+    /**
+     * Creates a new wrapper that represents a value of {@link RollingUpgradeModeChangeResult}.
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java]: We usually use "Default constructor, required for serialization." as javadoc.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+     */
+    public VisorRollingUpgradeChangeModeResult() {
+    }
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java]: Missing "// No-op." comment.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+    /**
+     * Creates a new wrapper that represents the given {@code changeModeRes}.
+     *
+     * @param changeModeRes status of the operation.
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java]: "status" -> "Status".

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+    private String targetVer;
+
+    /** Feature set that is supported by nodes. */
+    private Set<String> supportedFeatures;
+
+    /**
+     * Default constructor.
+     */
+    public VisorRollingUpgradeStatus() {
+    }
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java]: Missing "// No-op." comment.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatusResult.java [hunk 10]
```diff
@@ -88,21 +89,26 @@ public RollingUpgradeStatus status() {
...
+    public List<String> getUpdatedNodes() {
+        return updatedNodes;
     }
 
     /** {@inheritDoc} */
     @Override protected void writeExternalData(ObjectOutput out) throws IOException {
         out.writeObject(status);
-        out.writeObject(initVerNodes);
-        out.writeObject(updateVerNodes);
+        out.writeObject(initNodes);
```
Comment (akuznetsov-gridgain) [inline modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatusResult.java]: I think we have U.writeCollection() method?

**Review by akuznetsov-gridgain — CHANGES_REQUESTED** [review]

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java (hunk 1 above)
Comment (sk0x50) [reply modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeChangeModeResult.java]: Fixed.

File: modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java (hunk 2 above)
Comment (sk0x50) [reply modules/core/src/main/java/org/apache/ignite/internal/visor/ru/VisorRollingUpgradeStatus.java]: Thanks, fixed.

**Pull request closed without merging**